playbooks/gluster-cluster/gluster_volume.yml

```diff
@@ -3,7 +3,7 @@
 - name: Setup backend and create a GlusterFS cluster
   hosts: all
   remote_user: root
-  gather_facts: no
+  gather_facts: true
```

Comment:
I haven't understood the need for this - could you elaborate in commit message?

Reply:
Updated.

Reply:
Can we use fiter or subset option (as per https://docs.ansible.com/ansible/latest/modules/setup_module.html#options) so that not all facts need to be returned from remote hosts?

Reply:
I will explore if this can be used.